erts/emulator/nifs/win32/win_prim_file.c

```diff
@@ -291,14 +291,14 @@ static int is_ignored_name(int name_length, const WCHAR *name) {
     return 0;
 }
 
-static int get_drive_number(const efile_path_t *path) {
+static int get_drive_number(const WCHAR *path, int length) {
```

Comment:
I'd prefer that this and `is_executable_file` remain untouched, as `efile_path_t` is a promise that the path strictly follows the format used throughout the module. We've had a lot of issues with inconsistent path handling in the past so I'd like to follow convention whenever possible.

(These changes should be unnecessary if you repurpose `internal_read_link` as suggested earlier.)

Reply:
Yeah, I've already taken these changes back. I'm rebuilding to test with your previous suggestions now. I'm not sure if it's cygwin that does that, but grass grows faster than OTP builds in it...

Reply:
> I'm not sure if it's cygwin that does that, but grass grows faster than OTP builds in it...

I believe that mingw is faster in our experience, though not by much. It is all of the bash scripts in various places that makes it slow. They make it easier to maintain the build system on windows, but a lot slower to build.